app/containers/UIKit/MultiSelect/Items.tsx

```diff
@@ -46,13 +47,23 @@ const Item = ({ item, selected, onSelect }: IItem) => {
 	);
 };
 
+const RenderEmpty = () => {
+	const { colors } = useTheme();
+	return (
+		<View style={[styles.listEmptyContainer]}>
+			<Text style={[styles.noDataFound, { color: colors.titleText }]}>{I18n.t('No_results_found')}</Text>
```

Comment:
Use this https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/9109b8726912301b5b5e0fa159e814ad7a6c81e3/app/views/ThreadMessagesView/index.tsx#L486

Reply:
Only at this place or everywhere?